testing/mockeryutils/mockeryutils-types.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2024 The Nephio Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mockeryutils
+
+type MockHelper struct {
+	MethodName string
```

Comment:
It might be worth adding a comment to mention that this field maps to the `Method` field at https://godoc.org/github.com/stretchr/testify/mock#Call. This would help with code maintenance and for folks to get familiar with mockery.Also,  `ArgType` maps to `Arguments` field and the `RetArgList` maps to `ReturnArguments` field.